src/token.scss

```diff
@@ -92,12 +110,13 @@ $block: #{$fd-namespace}-token;
   @include fd-token-focus();
 
   &--compact {
-    padding-left: $fd-token-compact-padding-horizontal;
+    padding: 0 0 $fd-token-compact-padding-vertical $fd-token-compact-padding-horizontal;
     height: $fd-token-compact-height;
+    min-width: $fd-token-compact-min-width;
```

Comment:
Is this necessary to create a variable if we aren't using more than one time. because its just increases line of code.

Reply:
variables using rem values only once have now been removed